focuspoints.lrdevplugin/ExifUtils.lua

```diff
@@ -30,8 +31,7 @@ exiftoolWindows = LrPathUtils.child(exiftoolWindows, "exiftool.exe")
 
 function ExifUtils.getExifCmd(targetPhoto)
   local path = targetPhoto:getRawMetadata("path")
-  local metaDataFile = LrPathUtils.removeExtension(path)
-  metaDataFile = metaDataFile .. "-metadata.txt"
+  local metaDataFile = LrPathUtils.child(LrPathUtils.getStandardFilePath("temp"), LrUUID.generateUUID() .. ".txt")
```

Comment:
Not sure I understand.
LrPathUtils.child(LrPathUtils.getStandardFilePath("temp"), LrUUID.generateUUID() .. ".txt")
will save the file to the system TEMP directory which is always local and write enabled instead of the photo's current directory which may not be writable and is often on network.